### PR TITLE
Integrate more of the README.md into the sphinx site

### PR DESCRIPTION
**Pull request by GwnDaan**
Moved some of the more use case specific stuff from the main README, to the sphinx site to not overwhelm a first time viewer. While still allowing more detailed instructions in a central place on the site. 

I think adding the quick links and a catch phrase also looks nice at the top of our README, but I'd like thoughts on that. Replacing the 3 top images with a catchy demonstration video might be cool in the future? 🤔 

View modified readme [here](https://github.com/Open-Agriculture/AgIsoStack-plus-plus/blob/daan/cleanup-readme/README.md)

### CODE REVIEW

**Comment by GwnDaan** [comment]
Oh I also removed the minimum coverage and maximum duplication conditions from the quality gate, a bit more of the positive green checks didn't sound too bad to me :). We'll still see the coverage and duplication percentage in the overview posted by the bot

File: sphinx/source/Developer Guide.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+A default CAN driver plug-in will be selected for you based on your OS, but when compiling you can explicitly choose to use one of the natively supported CAN drivers by supplying the `CAN_DRIVER` variable.
+
+- :code:`-DCAN_DRIVER=SocketCAN` for Socket CAN support (This is the default for Linux)
+- :code:`-DCAN_DRIVER=WindowsPCANBasic` for the windows PEAK PCAN drivers (This is the default for Windows)
+- :code:`-DCAN_DRIVER=MacCANPCAN` for the MacCAN PEAK PCAN driver (This is the default for Mac OS)
+- :code:`-DCAN_DRIVER=TWAI` for the ESP TWAI driver (This is the preferred ESP32 driver)
+- :code:`-DCAN_DRIVER=MCP2515` for the MCP2515 CAN controller
+- :code:`-DCAN_DRIVER=WindowsInnoMakerUSB2CAN` for the InnoMaker USB2CAN adapter (Windows)
+- :code:`-DCAN_DRIVER=TouCAN` for the Rusoku TouCAN (Windows)
```
Comment (ad3154) [inline sphinx/source/Developer Guide.rst:66]: Need to add SysTec as well I think `SYS_TEC`

File: sphinx/source/Installation.rst [hunk 2]
```diff
@@ -26,55 +28,90 @@ We officially support building AgIsoStack++ from source on the following platfor
...
+      GIT_TAG        main
+   )
+   FetchContent_MakeAvailable(AgIsoStack)
    
-Non-CMake:
-^^^^^^^^^^
+   # Somewhere later in your CMakeLists.txt
+   target_link_libraries(<your executable name> PRIVATE isobus::Isobus isobus::HardwareIntegration isobus::Utility Threads::Threads)
+
+Now when you configure your CMake cache, the library will be pulled from GitHub and automatically made available for your project.
```
Comment (ad3154) [inline sphinx/source/Installation.rst:112]: We could possibly enhance this section a little to explain that selecting a specific commit may provide better stability and faster configuration times at the expense of not automatically updating 🤔. With a branch selected it has to fetch every time someone configures their CMake cache

**Review by ad3154 — DISMISSED** [review]
> I think adding the quick links and a catch phrase also looks nice at the top of our README, but I'd like thoughts on that. Replacing the 3 top images with a catchy demonstration video might be cool in the future?

I do really like your changes! The images are outdated anyways, I agree we could probably do something cooler.

File: sphinx/source/Installation.rst (hunk 2 above)
Comment (GwnDaan) [reply sphinx/source/Installation.rst:112]: I'll add that, good thought

File: sphinx/source/Developer Guide.rst (hunk 1 above)
Comment (GwnDaan) [reply sphinx/source/Developer Guide.rst:66]: Added 👍🏽

**Review by ad3154 — APPROVED** [review]

**Pull request merged**